cmake/Dependencies.cmake

```diff
@@ -393,19 +410,19 @@ if(USE_PYTORCH_QNNPACK)
     endif()
 
     if(NOT TARGET pytorch_qnnpack)
+      if(NOT USE_SYSTEM_PTHREADPOOL AND USE_INTERNAL_PTHREADPOOL_IMPL)
```

Comment:
Do we want PYTORCH_QNNPACK to have same behavior as XNNPACK? That is make it always use OSS? Does not have to be in this PR but it will require changes to our fork of QNNPACK. Maybe we can sync up and decide.

Reply:
The only downside as far as I can tell is that we would loose the ability to parallelize QNNPACK and NNPACK on Windows, where third-party pthreadpool is not supported.  If we are willing to leave Windows behind for NNPACK and QNNPACK we can do that.

Reply:
Actually coming to think about it, I think NNPACK and family do not build on Windows to begin with!

Reply:
QNNPACK works on Windows.  I don't think we care about *NNPACK performance on Windows builds.